docs/reference/settings/notification-settings.asciidoc

```diff
@@ -264,6 +264,13 @@ HTML feature groups>>.
 Set to `false` to completely disable HTML sanitation. Not recommended.
 Defaults to `true`.
 
+`xpack.notification.reporting.warning.kbn-csv-contains-formulas.text`::
+(<<dynamic-cluster-setting,Dynamic>>)
+Specifies a custom message to be sent if the formula verification criteria
+for CSV files, from {kib}'s {kib-ref}/reporting-settings-kb.html#reporting-csv-settings[`xpack.reporting.csv.checkForFormulas`], is true.
```

Comment:
```suggestion
for CSV files, from {kib}'s {kibana-ref}/reporting-settings-kb.html#reporting-csv-settings[`xpack.reporting.csv.checkForFormulas`], is true.
```

Reply:
@a03nikki, your syntax was near perfect! The attribute for Kibana is `{kibana-ref}`, which I added as a suggested change. For future reference, you can find the documentation attributes here: https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc